### ISSUE TITLE
PIL.UnidentifiedImageError: cannot identify image file .DS_Store

### ISSUE THREAD

**Issue by eddjrn** [d8ahazard/sd_dreambooth_extension#44]
**Platform**: M1 Mac OS , 32GB ram

When training I get the following error. The problem appears because finder generates that file automatically and is not an image. I do not know if also happens in Linux or Windows.

```
Error completing request
Arguments: ('pepe', '/Users/eduardoreyes/Desktop/backups/Training images/pepe/processed', '/Users/eduardoreyes/Desktop/backups/Training images/Stable-Diffusion-Regularization-Images-person_ddim/person_ddim', 'photo of pepe person', 'photo of a person', '', '', 1.0, 7.5, 40.0, 400, 512, False, True, 1, 1, 1, 1000, 1, True, 5e-06, False, 'constant', 0, False, 0.9, 0.999, 0.01, 1e-08, 1, 500, 500, 'no', True, '', False) {}
Traceback (most recent call last):
  File "/Users/eduardoreyes/Documents/stable-diffusion-webui/modules/ui.py", line 185, in f
    res = list(func(*args, **kwargs))
  File "/Users/eduardoreyes/Documents/stable-diffusion-webui/webui.py", line 54, in f
    res = func(*args, **kwargs)
  File "/Users/eduardoreyes/Documents/stable-diffusion-webui/extensions/sd_dreambooth_extension/dreambooth/dreambooth.py", line 256, in start_training
    trained_steps = main(config)
  File "/Users/eduardoreyes/Documents/stable-diffusion-webui/extensions/sd_dreambooth_extension/dreambooth/train_dreambooth.py", line 716, in main
    for step, batch in enumerate(train_dataloader):
  File "/Users/eduardoreyes/Documents/stable-diffusion-webui/env/lib/python3.10/site-packages/accelerate/data_loader.py", line 356, in __iter__
    next_batch = next(dataloader_iter)
  File "/Users/eduardoreyes/Documents/stable-diffusion-webui/env/lib/python3.10/site-packages/torch/utils/data/dataloader.py", line 630, in __next__
    data = self._next_data()
  File "/Users/eduardoreyes/Documents/stable-diffusion-webui/env/lib/python3.10/site-packages/torch/utils/data/dataloader.py", line 673, in _next_data
    data = self._dataset_fetcher.fetch(index)  # may raise StopIteration
  File "/Users/eduardoreyes/Documents/stable-diffusion-webui/env/lib/python3.10/site-packages/torch/utils/data/_utils/fetch.py", line 58, in fetch
    data = [self.dataset[idx] for idx in possibly_batched_index]
  File "/Users/eduardoreyes/Documents/stable-diffusion-webui/env/lib/python3.10/site-packages/torch/utils/data/_utils/fetch.py", line 58, in <listcomp>
    data = [self.dataset[idx] for idx in possibly_batched_index]
  File "/Users/eduardoreyes/Documents/stable-diffusion-webui/extensions/sd_dreambooth_extension/dreambooth/train_dreambooth.py", line 343, in __getitem__
    instance_image = Image.open(instance_path)
  File "/Users/eduardoreyes/Documents/stable-diffusion-webui/env/lib/python3.10/site-packages/PIL/Image.py", line 3186, in open
    raise UnidentifiedImageError(
PIL.UnidentifiedImageError: cannot identify image file '/Users/eduardoreyes/Desktop/backups/Training images/pepe/processed/.DS_Store'
```

**Comment by entmike** [comment]
Just delete `.DS_Store`?

**Comment by d8ahazard** [comment]
Fixed with b39403f
